src/Types/Image.php

```diff
@@ -51,11 +51,29 @@ public function toCData(RaylibFFIProxy $ffi): CData
 
     /**
      * @psalm-suppress MixedPropertyAssignment
+     * @psalm-suppress MixedInferredReturnType
      * @psalm-suppress MixedReturnStatement
      * @psalm-suppress MixedInferredReturnType
      */
     public function updateFromStruct(CData $imageStruct): CData
```

Comment:
I have the impression this method should not exist at all. It is not specific to `Image` struct types and only returns a C Address. If you agree, let's please replace all usages of `Image::updateFromStruct()` with `FFI::addr()` and delete this method.

If you don't agree, please just rename this method to something like `getStructAddress()` without receiving `CData` as a parameter.

Reply:
Removed method ```Image::updateFromStruct()``` and replace all method calls for using ```FFI::addr()```